rpc/src/cache_block_meta_service.rs

```diff
@@ -30,40 +30,39 @@ impl CacheBlockMetaService {
     ) -> Self {
         let thread_hdl = Builder::new()
             .name("solCacheBlkTime".to_string())
-            .spawn(move || loop {
-                if exit.load(Ordering::Relaxed) {
-                    break;
-                }
-                let recv_result = cache_block_meta_receiver.recv_timeout(Duration::from_secs(1));
-                match recv_result {
-                    Err(RecvTimeoutError::Disconnected) => {
+            .spawn(move || {
+                info!("CacheBlockMetaService has started");
+                loop {
+                    if exit.load(Ordering::Relaxed) {
                         break;
                     }
-                    Ok(bank) => {
-                        let mut cache_block_meta_timer = Measure::start("cache_block_meta_timer");
-                        Self::cache_block_meta(&bank, &blockstore);
-                        cache_block_meta_timer.stop();
-                        if cache_block_meta_timer.as_ms() > CACHE_BLOCK_TIME_WARNING_MS {
```

Comment:
what's the idea behind getting rid of this?

Reply:
This timer seemed like overkill for this service. If it takes more than 150ms to push < 50 bytes into the Blockstore, the system is going to fail in much more spectacular ways